docs/features/simple-upload-adapter.md

```diff
@@ -16,16 +16,16 @@ See the [Server–side configuration](#server-side-configuration) section to lea
 
 ## Installation
 
+<info-box info>
+	The {@link module:upload/adapters/simpleuploadadapter~SimpleUploadAdapter simple upload adapter} plugin is not available out–of–the–box in any of {@link builds/guides/overview#available-builds official editor builds}. You will need to create a custom editor build to use it. Check out the {@link builds/guides/integration/installing-plugins "Installing plugins"} guide to learn more.
```

Comment:
It's not true that you will have create a custom build (if you're using the editor from source). I'd remove that sentence and then it should be fine.

Reply:
When the architecture is so complex even maintainers get confused what is what :P